### PR TITLE
Modified getMaskForBranchCondition(...)

### PR DESCRIPTION
**Pull request by arianneb**
Modified getMaskForBranchCondition to return expected hex values
instead of expected hex values bit shifted by 4. Modified function
calls that were compensating for this bit shift.

Signed-off-by: Arianne Butler <arianne.butler@ibm.com>

### CODE REVIEW

File: compiler/z/codegen/InstOpCode.hpp [hunk 1]
```diff
@@ -214,7 +214,7 @@ getReverseBranchCondition(TR::InstOpCode::S390BranchCondition bc)
          return TR::InstOpCode::COND_BMR;
       default:
          {
-         uint8_t mask = (getMaskForBranchCondition(bc)>>4);
+         uint8_t mask = (getMaskForBranchCondition(bc));
```
Comment (fjeremic) [inline compiler/z/codegen/InstOpCode.hpp]: Unnecessary double parenthesis here.

**Comment by 0xdaryl** [comment]
Please fix the spelling of "Modified" in your commit and PR titles.

**Comment by arianneb** [comment]
Personal Build: http://vmfarm.ottawa.ibm.com/build_info.php?build_id=360755

**Comment by 0xdaryl** [comment]
Please remove the above comment referencing an IBM-internal build.

**Pull request merged**